save.go

```diff
@@ -93,6 +93,12 @@ func (d *InMAP) initFromCells(cells []*Cell, emis *Emissions, config *VarGridCon
 	for i, p := range config.CensusPopColumns {
 		d.popIndices[p] = i
 	}
+	d.mortIndices = make(map[string]int)
+	i := 0
+	for _, m := range config.MortalityRateColumns {
```

Comment:
The reason why it doesn't exactly mirror the code for creating `d.popIndices` is that `config.CensusPopColumns` is an array while `config.MortalityRateColumns` is a map.